### ISSUE TITLE
Bug: wrong-import-order

### ISSUE THREAD

**Issue by nickdrozd** [PyCQA/pylint#2175]
When I run Pylint against itself locally, I always see this message:

```
pylint/checkers/imports.py:38:0: C0411: third party import "import isort" should be placed before "import astroid" (wrong-import-order)
```

Why doesn't this trigger in CI? Maybe there's some ambiguity as to what qualifies as "third party". Running it locally I use `export PYTHONPATH=$PYTHONPATH:~/pylint:~/astroid`, and I guess other dependencies come through `pip`.

**Comment by PCManticore** [comment]
How is `astroid` installed, is it an editable installation or actually installed in `sys-packages`? This might all come from `isort`, since we use that to tell `pylint` what kind of import a particular package is (third party, local, first party etc).

**Comment by eight04** [comment]
It seems that `isort` thinks locally installed packages (`pip install -e .`) are not 3rd-party packages. This is inconvenient for library developers. We often test different libraries together by cloning them from Github and installing them locally (and maybe send a patch after some modifications).

**Comment by noyainrain** [comment]
See also #1797, which I think is related.

**Comment by Pierre-Sassoulas** [comment]
Closing as it's a problem from 2018 that I never saw. We probably fixed it with a proper isort configuration telling that astroid is a known third party..
